tidb-cloud/use-htap-cluster.md

```diff
@@ -11,7 +11,7 @@ With TiDB Cloud, you can create an HTAP cluster easily by specifying one or more
 
 > **Note:**
 >
-> A Developer Tier cluster has one TiFlash node by default and you cannot change the number.
+> Serverless Tier clusters have TiFlash support and can disable TiFlash.
```

Comment:
```suggestion
> Serverless Tier clusters support TiFlash. You can enable or disable TiFlash based on your needs.
```

Reply:
Oops, it was my bad. TiFlash is always enabled In Serverless Tier, users can not disable it.

Reply:
Can you take another look at it? I tried to fix it but not sure how to make it more concise and clear.